test/test_scaled_matmul_cuda.py

```diff
@@ -1234,6 +1286,199 @@ def test():
         else:
             test()
 
+    @unittest.skipIf(not PLATFORM_SUPPORTS_FP8 or IS_WINDOWS, f8_msg)
+    @unittest.skipIf(not IS_SM90, "cuBLAS blockwise scaling requires sm90+")
+    @unittest.skipIf(
+        _get_torch_cuda_version() < (12, 9),
+        "cuBLAS blockwise scaling added in CUDA 12.9",
+    )
+    @parametrize("output_dtype", [torch.bfloat16, torch.float32])
+    @parametrize("lhs_block,rhs_block", [(1, 1), (128, 1), (1, 128)])
+    @parametrize("M,N,K", [
+        # Nice size
+        (256, 768, 512),
+        # Requires padding for 128x128 scale
+        (384, 128, 1280),
+    ])
+    @parametrize("test_case", [
+        "x_eye_b_eye",
```

Comment:
seems like this case would never run, since it requires M == K == N?  Either add a supported shape or delete the test case?

Reply:
Missed this - added a supported shape (and WAR'd zero-block issues which arose)